src/svt-process-replies.js

```diff
@@ -0,0 +1,55 @@
+import { parseAnswers, match } from 'election-compass-match';
+
+// Based on https://github.com/svt/election-compass-match/
+
+const sortByRating = (a, b) => b.score - a.score;
+
+function parseAnswerToSVT(answers) {
+  const rangeSVTAnswer = answers
+    .map(answer => {
+      if (answer === '6') {
+        return '_';
+      }
+      return `${answer}/5`;
+    })
+    .join(';');
+  return parseAnswers(rangeSVTAnswer);
+}
+
+function parsePoliticalAnswerToSVT(answers) {
+  return answers
+    .split('')
+    .slice(0, 37) // this is only here temporarily to test old answers compared to the 37 new questions
+    .map(answer => {
+      // 6 means the user skipped it or didnt answer it
+      if (answer === '6') {
+        return null;
+      }
+      // SVT uses 4-level Likert scale by default while we use 5 level
+      // To make it work with the 5-level scale we need to set the type as range
+      if (Math.random() < 0.5) {
+        // Used to test out the scale
+        return { selectedIndex: answer - 1, isImportant: true, type: 'RANGE' };
+      }
+      return { selectedIndex: answer - 1, isImportant: false, type: 'RANGE' };
```

Comment:
What do you think about



```suggestion
      return { 
        selectedIndex: answer - 1,
        // Used to test out the scale
        isImportant: Math.random() < 0.5,
        type: 'RANGE' 
      };
```

Reply:
Fine by me, but we should remove this before we merge it to master, given we have the new questions ready